mkosi.md

```diff
@@ -543,6 +543,12 @@ a boolean argument: either "1", "yes", or "true" to enable, or "0",
   in building or further container import stages.  This option strips
   SELinux context attributes from the resulting tar archive.
 
+`Initrd=`, `--initrd`
```

Comment:
Should this maybe be `InitRd`? Also, should this support a directory like `mkosi.initrd`?

Reply:
We already have CacheInitrd=, so let's use the same naming here